docs/instance.md

```diff
@@ -31,9 +67,55 @@ Then, it creates some folders:
 - `/Documents/Music`, with the id `io.cozy.folders-music`
 - `/Documents/Videos`, with the id `io.cozy.folders-videos`
 
-The ids are forced to known values. So, even if these folders are moved or
-renamed, they can still be found for the permissions.
+**The ids are forced to known values :**  even if these folders are moved or
+renamed, they can still be found for the permissions. 
+
+**The names are localized :** If a locale is provided through the CLI, the folders will be created with 
+names in this locale. Otherwise, theses folders will be created in english and renamed to localized name the first time the locale is set (during onboarding).
+
+Then it creates the basic settings 
+
+- `owner_email` if an email was provided through the CLI
+- `locale` if a locale was provided through the CLI
+- `home_app` URL of the app to use as home, github.com/cozy/cozy-home by default
+- `onboarding_app` URL of the app to use as onboarding, github.com/cozy/cozy-onboarding by default
+- `market_url` URL of the market place to fetch manifests of installable applications.
```

Comment:
s/market/registry/ maybe? In my mind, the market/store is the app inside the cozy, and the registry is a list of available apps on a server.